network/bridge/vars.go

```diff
@@ -0,0 +1,18 @@
+package bridge
+
+const (
+	// DefaultBridge defines the default bridge name.
+	DefaultBridge = "p0"
+
+	// DefaultBridgeIP defines the default bridge ip.
+	DefaultBridgeIP = "192.168.0.1"
```

Comment:
Shall we change these subnet to other subnet, such as `172.17.0.1/24` like docker's.

Since there is too much possibility that `192.168.0.1/24` has conflict with user's own subnet of LAN.

WDYT?

Reply:
emmmmm, ok